dashboard/app/dsl/script_dsl.rb

```diff
@@ -129,7 +129,7 @@ def parse_output
       hidden: @hidden,
       wrapup_video: @wrapup_video,
       login_required: @login_required,
-      hideable_stages: @hideable_stages,
+      hideable_lessons: @hideable_lessons,
```

Comment:
if you go with option B from the other comment thread, you could update this code to populate both `hideable_lessons:` and `hideable_stages:` from `@hideable_lessons`, rather than having to go through and update all of the script files to contain both values (and then having to re-update them all again later to remove the old value)

Reply:
Good idea, I'll go with that.